### ISSUE TITLE
Fellowships minisite - build "application open" page

### ISSUE THREAD

**Issue by mmmavis** [mozilla/foundation.mozilla.org#1025]
WIP comps: https://projects.invisionapp.com/share/ZJ9YTAGEA#/screens/276734812_v2-FellowshipsUI_-_Apply-Applications_Open

/cc @taisdesouzalessa

**Comment by taisdesouzalessa** [comment]
This page will change pretty soon, in terms of structure and probably content. I had a conversation with @jessevondoom and @xmatthewx on that. 
So I would recommend holding on building that for now (focus on the other pages) until we have a solid state of this one :).

**Comment by mmmavis** [comment]
@taisdesouzalessa could you give me a ping once this page is ready for implementation? 😉

**Comment by taisdesouzalessa** [comment]
Sure! I don't have any status on this page at the moment. @jessevondoom or @alanmoo if you have any context about the next step for this page, could you please document it here?

**Comment by mmmavis** [comment]
@xmatthewx did you mean something like below?

![screenshot-2018-2-15 mozilla foundation 1](https://user-images.githubusercontent.com/2896608/36288752-256f0814-1271-11e8-9594-163ea9df854e.png)


---

@taisdesouzalessa I didn't include the big "Apply Now" block as shown in your Sketch file. That block is shown on multiple pages across the Fellowships site and I assume clicking "Apply Here" would bring user to the `/apply` page. That's why I don't think it makes much sense to include this component on the /apply page.

<img width="479" alt="screen shot 2018-02-15 at 4 56 42 pm" src="https://user-images.githubusercontent.com/2896608/36288804-71478f90-1271-11e8-9d7e-3168bd7f7943.png">

**Comment by xmatthewx** [comment]
@mmmavis yes. That layout is good. You can remove all the content—I don't think it came from the fellowship team. I'd rather have nothing there until they've finalized stuff on their end. 

Just have this: 
 - title: Be a Fellow
 - paragraph: Apply to be a fellow today!
 - button: Apply

And yes, you can leave out the component.

**Comment by taisdesouzalessa** [comment]
Agree with your suggestion @mmmavis :). 

@xmatthewx the content of the specific areas (Science, Open Web and so on) came from the fellowship team. The blurb above that (between "be a fellow" and "science") is a placeholder. Just wanted to flag that.

https://docs.google.com/a/mozillafoundation.org/document/d/1iiQ5lTFpAbEvRMYcNx9DqvaTmVjCE-AHwYh4PMtvveg/edit?disco=AAAABo0ZjxM

**Comment by xmatthewx** [comment]
Yep. That text suggests that a pserson can apply for any of those four focus areas when you hit the button below, but they can't. 

I want to shield @mmmavis from content in flux, carve the simplest path to getting the engineering done while we wait for all the content to be completed.

**Comment by taisdesouzalessa** [comment]
Thanks for the context, @xmatthewx! It makes sense to me :)

**Comment by mmmavis** [comment]
Gonna close this ticket since PR has been merged into the main `fellowships` branch. Once copy is ready we can update content on the page as a separate PR!
